src/colvarscript.h

```diff
@@ -88,6 +98,9 @@ class colvarscript  {
     cv_frame,
     cv_units,
     cv_colvar,
+    // Should the following subcommands be listed in the enum if they are handled by the cv_colvar function?
+    // do we want a mechanism to keep help strings for subcommands, or one big help string for the
```

Comment:
Just noticed this question.  In my view a separate help string for each subcommand is desirable for a more "permanent" implementation (i.e. one that is likely to stick around in a precompiled package for some time).  That would mean also breaking up `proc_colvar`, `proc_bias` and `proc_features` into individual commands.

Also, the implementation of `CVSCRIPT` that I added at this location in the branch `colvarscript-files` would also take care of any unused `enum` entries, because it generates them only if the corresponding scripting command is defined by an instance of `CVSCRIPT`.